### PR TITLE
Update docker-compose.yml file to fit Proximis 2.0 version

### PR DESCRIPTION
**Pull request by matthieupetel**

### CODE REVIEW

**Review by ker0x — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -8,15 +8,14 @@ services:
       - "80:80"
     volumes:
       - ~/www:/var/www/html
+      - ~/docker/proximis/nginx:/etc/nginx/conf.d
```
Comment (ajardin) [inline docker-compose.yml]: It's not needed if you keep the copy instruction in the Dockerfile.
https://github.com/EmakinaFR/docker-proximis/blob/proximis-2.0/nginx/Dockerfile#L5

File: docker-compose.yml [hunk 2]
```diff
@@ -33,3 +32,10 @@ services:
       - "9200:9200"
       - "9300:9300"
     tty: true
+
+  redis:
+    image: redis:latest
+    env_file: docker-env
+    ports:
+      - "6379:6379"
+    tty: true
```
Comment (ajardin) [inline docker-compose.yml]: Could you add an extra line at the end of the file? :)

File: docker-compose.yml (hunk 1 above)
Comment (ker0x) [reply docker-compose.yml]: Fix

File: docker-compose.yml (hunk 2 above)
Comment (ker0x) [reply docker-compose.yml]: Fix

**Pull request merged**